store.go

```diff
@@ -33,13 +34,15 @@ var (
 type RepoStore struct {
 	repositories map[string]*AsyncRepoCloner
 	mutex        sync.RWMutex
+	repoPath     string
 }
 
 // NewRepoStore initializes a new RepoStore.
-func NewRepoStore() *RepoStore {
+func NewRepoStore(repoPath string) *RepoStore {
```

Comment:
Do we have any external users that might be caught out by this function now having an argument?

Reply:
It's not expected to be used by many people apart from with Faros. Since we haven't cut a 1.0 release, I'm not too concerned about changing this